llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -769,10 +769,10 @@ class LoadInst final : public Instruction {
   unsigned getNumOfIRInstrs() const final { return 1u; }
   static LoadInst *create(Type *Ty, Value *Ptr, MaybeAlign Align,
                           Instruction *InsertBefore, Context &Ctx,
-                          const Twine &Name = "");
+                          bool isVolatile = false, const Twine &Name = "");
```

Comment:
Ah you need to capitalize `isVolatile` to `IsVolatile` to comply with LLVM's coding style.